client/dashboard/sites/overview-plan-card/index.tsx

```diff
@@ -24,6 +42,29 @@ export default function PlanCard( { site }: { site: Site } ) {
 	const { data: mediaStorage, isLoading: isLoadingMediaStorage } = useQuery(
 		siteMediaStorageQuery( site.ID )
 	);
+	const { startInSeconds, endInSeconds } = getCurrentMonthRangeTimestamps();
+	const { data: bandwidth, isLoading: isLoadingBandwidth } = useQuery( {
+		...siteMetricsQuery( site.ID, {
+			start: startInSeconds,
+			end: endInSeconds,
+			metric: 'response_bytes_persec',
+		} ),
+		enabled: !! site.is_wpcom_atomic,
+		select: ( data ) => {
+			if ( ! data ) {
+				return data;
+			}
+			const domain = getSiteDisplayUrl( site );
+			return data.data.periods.reduce(
+				( acc, curr ) => acc + ( curr.dimension[ domain ] || 0 ),
+				0
+			);
```

Comment:
I could probably do all this junk in the query layer, and keep "monthly-bandwidth" under its own cache key. But I wanted to see whether we would change the meaning of "monthly bandwidth" first.